neo/rawio/neuralynxrawio/neuralynxrawio.py

```diff
@@ -189,17 +191,30 @@ def _parse_header(self):
             for idx, chan_id in enumerate(chan_ids):
                 chan_name = chan_names[idx]
 
-                chan_uid = (chan_name, chan_id)
+                chan_uid = (chan_name, str(chan_id))
                 if ext == 'ncs':
+                    file_mmap = self._get_file_map(filename)
+                    n_packets = copy.copy(file_mmap.shape[0])
```

Comment:
why do we need copy ??
Maybe `n_packets = int(file_mmap.shape[0])` ?

Reply:
Hm, yes, this seem not necessary. I will update this.
```suggestion
                    n_packets = int(file_mmap.shape[0])
```